### PR TITLE
Run formatter over playback controller

### PR DESCRIPTION
**Pull request by DavidFair**
**Changes**

Runs the Android studio formatter over the playback controller. This
should make it easier to use automated tooling and clear up some diffs (#1302)
which have touched this file too.

### CODE REVIEW

**Review by mueslimak3r — APPROVED** [review]
Thanks for tidying this up 👍

File: app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java [hunk 1]
```diff
@@ -176,14 +176,31 @@ public MediaSourceInfo getCurrentMediaSource() {
...
+    public boolean isLiveTv() {
+        return isLiveTv;
+    }
+
+    public int getSubtitleStreamIndex() {
+        return (mCurrentOptions != null && mCurrentOptions.getSubtitleStreamIndex() != null) ? mCurrentOptions.getSubtitleStreamIndex() : -1;
+    }
+
+    public @Nullable
+    Integer getAudioStreamIndex() {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java:196]: I don't think this should be multiline?

```suggestion
    public @Nullable Integer getAudioStreamIndex() {
```

Comment (DavidFair) [reply app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java:196]: The formatter will split it back into multi line
Would the following be better?
```
    @Nullable
    public Integer getAudioStreamIndex() {
```

Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java:196]: Yeah that would be better

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**